ote/src/clj/ote/services/taxiui_service.clj

```diff
@@ -104,6 +104,7 @@
 
 (defn fetch-pricing-statistics
   [db {:keys [sorting filters]}]
+  ; TODO auth checks
```

Comment:
Auth checks seem to be missing here.

Reply:
And on line 125 as well.

Reply:
Funnily enough, the auth check is in the actual API endpoint definition below, just forgot to remove this TODO :) Too much coffee?